data/json/monsters/mammal.json

```diff
@@ -119,6 +119,78 @@
     "special_attacks": [ [ "EAT_FOOD", 120 ] ],
     "flags": [ "SEES", "SMELLS", "HEARS", "WARM", "SWIMS", "ANIMAL", "PATH_AVOID_DANGER_1" ]
   },
+  {
+    "id": "mon_boar_wild_piglet",
+    "type": "MONSTER",
+    "name": "wild boar piglet",
+    "description": "Originally not native in the US, this wild omnivore is a crossbreed between the Eurasian wild boar escaped from hunting reserves and the domesticated pig went feral in the wilderness.  Its population has skyrocketed in the US during the last few decades.",
+    "default_faction": "pig",
+    "bodytype": "pig",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "MAMMAL" ],
+    "volume": "10000 ml",
+    "weight": "10 kg",
+    "hp": 5,
+    "speed": 80,
+    "material": [ "flesh" ],
+    "symbol": "b",
+    "color": "brown",
+    "looks_like": "mon_pig_piglet",
+    "aggression": -10,
+    "morale": 10,
+    "melee_skill": 1,
+    "melee_dice": 1,
+    "melee_dice_sides": 2,
+    "melee_cut": 1,
+    "dodge": 4,
+    "harvest": "mammal_small_fur",
+    "path_settings": { "max_dist": 10 },
+    "fear_triggers": [ "SOUND", "PLAYER_CLOSE" ],
+    "placate_triggers": [ "MEAT" ],
+    "death_function": [ "NORMAL" ],
+    "upgrades": { "age_grow": 38, "into": "mon_boar_wild" },
+    "biosignature": { "biosig_item": "feces_manure", "biosig_timer": 7 },
+    "special_attacks": [ [ "EAT_FOOD", 40 ] ],
+    "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM", "KEENNOSE", "BLEED" ]
+  },
+  {
+    "id": "mon_boar_wild",
+    "type": "MONSTER",
+    "name": "wild boar",
+    "description": "Originally not native in the US, this wild omnivore is a crossbreed between the Eurasian wild boar escaped from hunting reserves and the domesticated pig went feral in the wilderness.  Its population has skyrocketed in the US during the last few decades.",
+    "looks_like": "mon_pig",
+    "default_faction": "pig",
+    "bodytype": "pig",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "MAMMAL" ],
+    "volume": "100000 ml",
```

Comment:
I guess this could be converted to liters for convenience.

Reply:
Ok, changed it to liters.